packages/sdk/docs/functions/getDepositLogs.md

```diff
@@ -0,0 +1,17 @@
+[@across-protocol/integrator-sdk](../globals.md) / getDepositLogs
+
+# Function: getDepositLogs()
+
+> **getDepositLogs**(`__namedParameters`): `undefined` \| `Log`\<`bigint`, `number`, `false`, `undefined`, `true`, readonly [`object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`, `object`], `"V3FundsDeposited"`\>
```

Comment:
We can probably use the generic Log type here, but if we build these docs again during CI then this will be overwritten?

Reply:
Yea, we'd need to explicitly type the return type of `getDepositLog` via TypeScript